### PR TITLE
[AGPUSH-2156][CHANGE] Rename HttpRequestUtil.extractSortingQueryParam…

### PR DESCRIPTION
**Pull request by diogoalbuquerque**
…Value method

- Removed extractSortingQueryParamValue method from HttpRequestUtil.java
- Created CommonUtils.java and added isAscendingOrder method
- Moved the ascending order tests from HttpRequestHelperTest.java to CommonsUtilsTest.java

### CODE REVIEW

**Comment by matzew** [comment]
taking a :eye: on this

**Comment by matzew** [comment]
Thanks dude! 

:tada:

File: jaxrs/src/main/java/org/jboss/aerogear/unifiedpush/rest/util/CommonUtils.java [hunk 1]
```diff
@@ -0,0 +1,15 @@
+package org.jboss.aerogear.unifiedpush.rest.util;
```
Comment (dimitraz) [inline jaxrs/src/main/java/org/jboss/aerogear/unifiedpush/rest/util/CommonUtils.java]: do you mind adding license headers here?

File: jaxrs/src/test/java/org/jboss/aerogear/unifiedpush/rest/util/CommonUtilsTest.java [hunk 2]
```diff
@@ -0,0 +1,28 @@
+package org.jboss.aerogear.unifiedpush.rest.util;
```
Comment (dimitraz) [inline jaxrs/src/test/java/org/jboss/aerogear/unifiedpush/rest/util/CommonUtilsTest.java]: Same as above

**Comment by matzew** [comment]
@dimitraz I did that already:

https://github.com/aerogear/aerogear-unifiedpush-server/commit/2813d3a023c169c6f7c9c190aee2ab22d270d349

**Comment by dimitraz** [comment]
@matzew oh ok 👍

**Comment by matzew** [comment]
@dimitraz :tada: :beer:

**Pull request merged**